### ISSUE TITLE
[review-expectopatronum] References

### ISSUE THREAD

**Issue by martinju** [NorskRegnesentral/shapr#178]
Hi @expectopatronum
Just opening an issue to handle the reference comments from https://github.com/openjournals/joss-reviews/issues/2027#issuecomment-581468027

- [x] 1. Lundberg, S. (2019): I am not sure if this is correct/necessary. On their Github page they state which references should be used, this one doesn't seem to be there. (in case this actually should be there, I think SHAP should be capitalized)
- [x] 2. Maybe this also needs to be cited (very recent): https://raw.githubusercontent.com/slundberg/shap/master/docs/references/tree_explainer.bib
- [x] 3. Pedersen & Benesty: Title should be "lime: Local Interpretable Model-Agnostic Explanations"
- [x] 4. Furthermore, capitalization of the titles and conference names is not consistent (some are capitalized, others are all lower case).

**Comment by martinju** [comment]
My attempt to fix the above issues can be found in PR #179. @expectopatronum please let me know if you find this satisfactory. The pdf can be found here: https://github.com/NorskRegnesentral/shapr/blob/references/inst/joss_paper/paper.pdf

Comments per issue:

1. I have removed this reference, and now rather mention the name of the Python package.
2. From my understanding the relevant part of this paper applies the TreeSHAP method already mentioned and cited (but now call it TreeExplainer instead). However, since this is a proper journal paper, I think I think it is a good idea to include it.
3. Fixed
4. I went through all citations and fixed uppercase/lowercase issues.

**Comment by expectopatronum** [comment]
1, 2, 3 look good!
4. there is still a mix of lower case and capitalization for conference/journal names

**Comment by nikolase90** [comment]
@martinju There is a mix of capital and lower cases where we cite R? Could you fix that?

**Comment by martinju** [comment]
I found no specific guidelines on how to format the BibTex entries for JOSS, but now enforced a strict convention of using uppercase for the first word in paper titles/journals and after colons + abbreviations like AI. For programming languages and packages I used the convention enforced by the authors (i.e. R, C++, Rcpp, data.table). Everything is all
lowercase, no matter the convention used by the journals themselves. 

For citing R I had used the bibTeX entry provided by R itself, running citation():

  @Manual{,
    title = {R: A Language and Environment for Statistical Computing},
    author = {{R Core Team}},
    organization = {R Foundation for Statistical Computing},
    address = {Vienna, Austria},
    year = {2019},
    url = {https://www.R-project.org/},
  }

In the latest version I deleted the address field and force lowercased the organization (and title). R Core Team is the author, so that is left unchanged.

@expectopatronum I am sorry to ask you to look over the references once again here: https://github.com/NorskRegnesentral/shapr/blob/references/inst/joss_paper/paper.pdf
I sincerely hope this is satisfactory. If not, I kindly ask you to pinpoint the exact issues.
